src/components/Admin/ListSkills/ListSkills.js

```diff
@@ -209,26 +219,174 @@ class ListSkills extends React.Component {
                     record.group,
                     record.language,
                     record.reviewStatus,
-                    record.editStatus,
                     record.skillTag,
                   )
                 }
               >
                 Edit
+              </span>
+              <span style={{ marginLeft: '5px', marginRight: '5px' }}> | </span>
+              <span
+                style={{ cursor: 'pointer', color: '#49A9EE' }}
+                onClick={() =>
+                  this.handleDelete(
+                    record.skillName,
+                    record.model,
+                    record.group,
+                    record.language,
+                  )
+                }
+              >
+                Delete
+              </span>
+            </span>
+          );
+        },
+      },
+    ];
+
+    this.delColumns = [
+      {
+        title: 'Name',
+        dataIndex: 'skillName',
+        width: '20%',
+      },
+      {
+        title: 'Model',
+        dataIndex: 'model',
+        width: '10%',
+      },
+      {
+        title: 'Group',
+        dataIndex: 'group',
+        filters: [
```

Comment:
@PrP-11 Please fetch these groups from `https://api.susi.ai/cms/getGroups.json` rather than hard-coding them so that if these groups are changed in server in future then they would change here as well.